clients/src/main/java/org/apache/kafka/clients/consumer/internals/PrototypeAsyncConsumer.java

```diff
@@ -469,17 +474,19 @@ public void commitSync(Map<TopicPartition, OffsetAndMetadata> offsets) {
 
     @Override
     public void commitSync(Map<TopicPartition, OffsetAndMetadata> offsets, Duration timeout) {
-        CompletableFuture<Void> commitFuture = commit(offsets);
+        CompletableFuture<Void> commitFuture = wakeupTrigger.setActiveTask(commit(offsets));
```

Comment:
Do we need to call `setActiveTask` before the event is added to eventHandler like we did in `committed()`?